packages/webapp/public/locales/en/translation.json

```diff
@@ -161,7 +161,9 @@
     "ANIMAL_ID": "ID",
     "ANIMAL_IDENTIFICATION": "Identification",
     "ANIMAL_LOCATIONS": "Location",
-    "ANIMAL_TYPE": "Type"
+    "ANIMAL_TYPE": "Type",
+    "SEARCH_INVENTORY_PLACEHOLDER": "Search your inventory",
+    "SEARCH_RESULTS_TEXT": "Showing {{count}} results"
```

Comment:
I wonder if we need to do something like this:
```
"SPECIES_COUNT_one": "{{count}} species",
"SPECIES_COUNT_other": "{{count}} species",
```

Reply:
Cool added that! Thanks

Reply:
Sorry Duncan, the library handles this. I should've shared this 🙏 
https://www.i18next.com/translation-function/plurals#singular-plural

Reply:
ooo purdy!